package.json

```diff
@@ -37,6 +37,7 @@
     "faker": "^3.1.0",
     "fibers": "^1.0.14",
     "font-awesome": "^4.6.3",
+    "google-libphonenumber": "^2.0.11",
```

Comment:
I didn't add that intentionally. It must have been a mistake while working with the multivendor task